spec/operators/distinct-spec.ts

```diff
@@ -1,219 +1,215 @@
-    const values = {a: 1, b: 2, c: 3, d: 4, e: 5, f: 6};
-    const e1 =   hot('--a--b--c--d--e--f--|', values);
-    const e1subs =   '^                   !';
-    const expected = '--a--b--c-----------|';
-    const selector = (value: number) => value % 3;
-
-    expectObservable((<any>e1).distinct(selector)).toBe(expected, values);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-  });
-
-  it('should raises error when selector throws', () => {
-    const e1 =   hot('--a--b--c--d--e--f--|');
-    const e1subs =   '^          !         ';
-    const expected = '--a--b--c--#         ';
-    const selector = (value: string) => {
-      if (value === 'd') {
-        throw new Error('d is for dumb');
-      }
-      return value;
-    };
-
-    expectObservable((<any>e1).distinct(selector)).toBe(expected, undefined, new Error('d is for dumb'));
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-  });
-
-  it('should support a flushing stream', () => {
-    const e1 =   hot('--a--b--a--b--a--b--|');
-    const e1subs =   '^                   !';
-    const e2 =   hot('-----------x--------|');
-    const e2subs =   '^                   !';
-    const expected = '--a--b--------a--b--|';
-
-    expectObservable((<any>e1).distinct(null, e2)).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-    expectSubscriptions(e2.subscriptions).toBe(e2subs);
-  });
-
-  it('should raise error if flush raises error', () => {
-    const e1 =   hot('--a--b--a--b--a--b--|');
-    const e1subs =   '^            !';
-    const e2 =   hot('-----------x-#');
-    const e2subs =   '^            !';
-    const expected = '--a--b-------#';
-
-    expectObservable((<any>e1).distinct(null, e2)).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-    expectSubscriptions(e2.subscriptions).toBe(e2subs);
-  });
-
-  it('should unsubscribe from the flushing stream when the main stream is unsubbed', () => {
-    const e1 =   hot('--a--b--a--b--a--b--|');
-    const e1subs =   '^          !         ';
-    const e2 =   hot('-----------x--------|');
-    const e2subs =   '^          !         ';
-    const unsub =    '           !         ';
-    const expected = '--a--b------';
-
-    expectObservable((<any>e1).distinct(null, e2), unsub).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-    expectSubscriptions(e2.subscriptions).toBe(e2subs);
-  });
-
-  it('should allow opting in to default comparator with flush', () => {
-    const e1 =   hot('--a--b--a--b--a--b--|');
-    const e1subs =   '^                   !';
-    const e2 =   hot('-----------x--------|');
-    const e2subs =   '^                   !';
-    const expected = '--a--b--------a--b--|';
-
-    expectObservable((<any>e1).distinct(null, e2)).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-    expectSubscriptions(e2.subscriptions).toBe(e2subs);
-  });
+import * as Rx from '../../src/Rx';
```

Comment:
Another one were git is freaking out... figure out what went wrong if you can.

Reply:
CLRF vs LF